crates/vm/levm/src/constants.rs

```diff
@@ -2,11 +2,13 @@ use ethrex_core::{H256, U256};
 
 pub const WORD_SIZE_IN_BYTES: U256 = U256([32, 0, 0, 0]);
 pub const WORD_SIZE_IN_BYTES_USIZE: usize = 32;
+pub const WORD_SIZE_IN_BYTES_U64: u64 = 32;
 
 pub const SUCCESS_FOR_CALL: i32 = 1;
 pub const REVERT_FOR_CALL: i32 = 0;
 pub const HALT_FOR_CALL: i32 = 2;
-pub const SUCCESS_FOR_RETURN: i32 = 1;
+pub const SUCCESS_FOR_RETURN: u8 = 1;
+pub const REVERT_FOR_RETURN: u8 = 0;
```

Comment:
These variable names are not very accurate. I was about to suggest to rename them but I think we don't even need them in the first place though.

Reply:
Changed in [Remove success/revert constants](https://github.com/lambdaclass/ethrex/pull/1454/commits/2e8be8f38952c5ca9c147558d3f9376a384f116c)